kymatio/frontend/torch_frontend.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABCMeta, abstractmethod
+import torch
+import torch.nn as nn
+from packaging import version
+
+assert version.parse(torch.__version__)>=version.parse("1.3"), 'Current PyTorch version is ' + str(torch.__version__) +\
```

Comment:
Same

Reply:
same - I however have a proposal!

Reply:
cf proposing later to build a backend for a given version of tf(or even python)

Reply:
Ah yes, sure. These are frontends, though. It should be easier to have those be more compatible (since they're doing less heavy lifting).

Reply:
no. I had some bugs in the frontend because APIs have changed.

Reply:
All right then. In TF or torch?

Reply:
TF - not torch(or not yet) I'm ok to give a short to torch, but TF was really a pain..